website/docs/sys-mgmt/background-tasks.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Background tasks
+slug: /background-tasks
+---
+
+authentik uses background tasks to run various operations independently and asynchronously from the web requests it receives. Those tasks are run by the [worker](./ops/worker.md).
+
+## What are background tasks used for?
+
+Here is a non-exhaustive list of what background tasks are used for:
+
+- outposts: manage [Outposts](../add-secure-apps/outposts/index.mdx) deployments, send notifications to outpost when a refresh is needed
+- housekeeping: clean up old objects, check for updates, etc.
+- blueprints: import and apply [Blueprints](../customize/blueprints/index.mdx)
+- synchronization: sync users to and from authentik, from sources and to providers. This is used by:
+    - [SCIM Provider](../add-secure-apps/providers/scim/index.md)
+    - [Google Workspace Provider](../add-secure-apps/providers/gws/index.md)
+    - [Microsoft Entra Provider](../add-secure-apps/providers/entra/index.md)
+    - [SSF Provider](../add-secure-apps/providers/ssf/index.md)
+    - [Kerberos Source](../users-sources/sources/protocols/kerberos/index.md)
+    - [LDAP Source](../users-sources/sources/protocols/ldap/index.md)
+- enterprise [license management](../enterprise/manage-enterprise.mdx#license-management)
+- notifications: send [Notifications](./events/notifications.md) when events are created
+- emails: send emails when triggered by one of the email stages or otherwise
+
+## Schedules
+
+Some tasks run on a schedule. Those schedules can be [configured](#schedule-configuration). Schedules can be manually triggered by clicking the play arrow.
+
+## Tasks statuses
+
+A task can have the following statuses:
+
+- **Successful**: the task executed successfully. No extra action is required.
+- **Warning**: the task emitted a warning. Look at the task logs for further information. See [Failed tasks](#failed-tasks) for more details.
+- **Error**: the task failed to process. Either the task threw an exception, or reported an other error. Look at the task logs for further information. See [Failed tasks](#failed-tasks) for more details.
+- **Waiting to run**: the task has been queued for running, but no worker picked it up yet, either because none are available, they are already busy, or because it's just been queued.
```

Comment:
```suggestion
- **Waiting to run**: the task has been queued for running, but no worker has picked it up yet, either because none are available, they are already busy, or because it's just been queued.
```